repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,378 @@
+          "default" : 1000
+        },
+        "leader-proxy-connection-timeout" : {
+          "description" : "Maximum time, in milliseconds, to wait for connecting to the current Marathon leader from another Marathon instance.",
+          "type" : "integer",
+          "default" : 5000
+        },
+        "leader-proxy-read-timeout" : {
+          "description" : "Maximum time, in milliseconds, for reading from the current Marathon leader.",
+          "type" : "integer",
+          "default" : 10000
+        },
+        "local-port-max" : {
+          "description" : "Max port number to use when assigning globally unique service ports to apps.",
+          "type" : "integer",
+          "default" : 20000
+        },
+        "local-port-min" : {
+          "description" : "Min port number to use when assigning globally unique service ports to apps.",
+          "type" : "integer",
+          "default" : 10000
+        },
+        "logging-level" : {
+          "description" : "Set logging level to one of: off, error, warn, info, debug, trace, all",
+          "type" : "string"
+        },
+        "marathon-store-timeout" : {
+          "description" : "(deprecated) Maximum time, in milliseconds, to wait for persistent storage operations to complete. This option is no longer used and will be removed in a later release.",
+          "type" : "integer"
+        },
+        "master" : {
+          "description" : "The URL of the Mesos master",
+          "type" : "string"
+        },
+        "max-apps" : {
+          "description" : "The maximum number of applications that may be created.",
+          "type" : "integer"
+        },
+        "max-tasks-per-offer" : {
+          "description" : "Maximum tasks per offer. Do not start more than this number of tasks on a single offer.",
+          "type" : "integer",
+          "default" : 100
+        },
+        "mesos-authentication-principal" : {
+          "description" : "Mesos Authentication Principal.",
+          "type" : "string"
+        },
+        "mesos-authentication-secret-file" : {
+          "description" : "Mesos Authentication Secret.",
+          "type" : "string"
+        },
+        "mesos-leader-ui-url" : {
+          "description" : "The host and port (e.g. \"http://mesos_host:5050\") of the Mesos master.",
+          "type" : "string",
+          "default" : "/mesos"
+        },
+        "mesos-role" : {
+          "description" : "Mesos role for this framework. If set, Marathon receives resource offers for the specified role in addition to resources with the role designation '*'.",
+          "type" : "string"
+        },
+        "mesos-user" : {
+          "description" : "Mesos user for this framework.",
+          "type" : "string",
+          "default" : "matthias"
+        },
+        "metrics" : {
+          "description" : "Enabled: (Default) Enable metric measurement of service method calls. Disabled: Disable metric measurement of service method calls.",
```

Comment:
Oh, I see what you're doing now. Okay, you can ignore this and the previous comment.